epochStart/mock/requestHandlerFactoryMock.go

```diff
@@ -0,0 +1,25 @@
+package mock
+
+import (
+	"github.com/multiversx/mx-chain-go/dataRetriever/requestHandlers"
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/testscommon"
+)
+
+// RequestHandlerFactoryMock -
+type RequestHandlerFactoryMock struct {
```

Comment:
This is not being used anywhere + there is already another identical mock like this in `testscommon/factory`

Reply:
deleted